These Updated Should Include Several Fixes

Lhallin opened the pull request:
This is our latest iteration  fixed and added files along with some changes

Lhallin committed:
Updated Index.html, CSS and Added Images!

Lhallin committed:
Updated Files and Images

Added a New iteration of our pages.   Previous issues has been broken images or 

missing images and folders with images.  All these should be fixed in this iteration.

and I am opening a pull request for merging into our team leads project main.

Lhallin committed:
Merge pull request #3 from Lhallin/The-Library

Updated Files Fixed Broken Images

IamAudra commented:
Please change the `Index.html` to `index.html`.  Please address the `style.css` conflicts.

Lhallin committed:
Added  Updated index and style Files

The are the Index.html  and Style.css files.

Hoping that these updated files will solve vercel issues.

Lhallin commented:
Because of conflicts.
